### ISSUE TITLE
False positive on Enum.__members__.items()

### ISSUE THREAD

**Issue by daneah** [PyCQA/pylint#4123]
### Steps to reproduce
1. Create an `Enum` subclass
2. Try to iterate over the Enum contents using `MyEnum.__members__.items()`
3. Lint the code

### Current behavior
observe `E1101: Function '__members__' has no 'items' member (no-member)`

### Expected behavior

Iterating over an Enum [per the Python docs](https://docs.python.org/3/library/enum.html#iteration) does not result in a linting error.

### pylint --version output
```
pylint 2.7.0
astroid 2.5
```

**Comment by ikraduya** [comment]
I can make a patch about this issue

**Comment by postlund** [comment]
I'm facing the same issue, but with `values` instead of `items`. Would expect it's the same for `keys` as well.

**Comment by VladimirSlavik** [comment]
I have `__members__[something]` which gives me `unsubscriptable-object`. I guess everything under `__members__` is invisible...
